pkg/ottl/ottlfuncs/README.md

```diff
@@ -1370,6 +1370,44 @@ Examples:
 
 - `UnixSeconds(Time("02/04/2023", "%m/%d/%Y"))`
 
+### URI
```

Comment:
Should this actually be `URL`? All of the semantic conventions are specifically for URLs instead of URIs, and we are requiring `domain` (so a filesystem URI wouldn't work).